config/cypress.yml

```diff
@@ -9,7 +9,7 @@ default_bundle: "2.7.0"
 enable_logging: false
 
 #set to true to enable a banner at the top of every page with the "banner_message" text from below
-banner: false
+banner: <%= ENV["CYPRESS_BANNER"] || false %>
 banner_message: "This server is for demonstration purposes; data on it will be removed every Saturday at 11:59 PM Eastern Time"
```

Comment:
Is banner supposed to be separate from `banner_message`? Do we want `banner` configurable from the admin dashboard?